.github/workflows/test.yml

```diff
@@ -89,24 +102,30 @@ jobs:
       fail-fast: false
     name: Expo Webpack
     steps:
-      - uses: actions/cache@v2
+      - name: 🗂 Download builds
+        uses: actions/cache@v2
         with:
           path: '*'
           key: v2-${{ github.sha }}-${{ matrix.node }}
-      - name: Install puppeteer
+      - name: 🏗 Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
```

Comment:
```suggestion
          node-version: ${{ matrix.node }}
```

Reply:
No, this one is intentional. The puppeteer job is a separate unrelated one to the normal testing. We don't have a test on multiple node versions for that now, and I don't think it would add much value if we add it.